packages/subgraph/src/entities/Global.ts

```diff
@@ -26,6 +26,8 @@ export function getGlobal(): Global {
     newGlobal.totalNumberOfTroves = 0;
     newGlobal.rawTotalRedistributedCollateral = BIGINT_ZERO;
     newGlobal.rawTotalRedistributedDebt = BIGINT_ZERO;
+    newGlobal.totalNumberOfLqtyStakes = 0;
+    newGlobal.totalLqtyTokensStaked = DECIMAL_ZERO;
```

Comment:
Why do we use `DECIMAL_ZERO` instead of `BIGINT_ZERO`?

Reply:
Hmm I didn't think about it too much but wouldn't we want the most granular storage for fractions of tokens? A user could deposit 0.232382948934854 LQTY right?

Reply:
Hm, I don’t remember how we handle this in the graph, but big ints already include 18 decimal places. I see other amounts here, like `rawTotalRedistributedCollateral` that may have decimals too and are using big int. :thinking:

Reply:
@danielattilasimon how come we don't use BigInt for all the values? I followed stability pool approach which converts the event data to a BigDecimal

Reply:
BigDecimals have a lot of precision, too. I used them over BigInt for "ergonomic" reasons. If I'm tinkering with queries against a subgraph using GraphiQL, I'd much rather be writing queries like this:
```graphql
{
  troves(where: {collateral_gt: 99.99}) {
    collateral
    debt
  }
}
```
as opposed to this:
```
{
  troves(where: {collateral_gt: 99990000000000000000}) {
    collateral
    debt
  }
}
```

The reason I added the raw* fields to the Trove entity is because frontends may have to perform additional calculations on those values to get the most up-to-date collateral/debt/ICR/... values, in case there have been liquidations by redistribution since the last Trove modification. It seemed like a good idea to retain all the precision for these for that reason.

However, I'd also be fine with using BigInt everywhere, if that seems simpler.

Reply:
Ah of course! No it’s much better UX to decimalize the ints. Thanks @danielattilasimon